newIDE/app/src/UI/User/UserChip.js

```diff
@@ -27,44 +31,56 @@ const styles = {
 };
 
 type Props = {|
-  profile: ?Profile,
-  onClick: () => void,
-  displayNotificationBadge: boolean,
+  onOpenProfile: () => void,
 |};
 
-const UserChip = ({ profile, onClick, displayNotificationBadge }: Props) => {
+const UserChip = ({ onOpenProfile }: Props) => {
+  const authenticatedUser = React.useContext(AuthenticatedUserContext);
+  const { profile, onLogin, onCreateAccount, loginState } = authenticatedUser;
+  const displayNotificationBadge = hasPendingNotifications(authenticatedUser);
   const classes = useStyles();
-  return (
+  return loginState === 'loggingIn' ? (
+    <CircularProgress size={20} />
```

Comment:
Should it be in a `Line` so that it keeps an height similar to a button?

Reply:
The Line wasn't giving the expected height for some reason, so I just changed the size to match the same height 👍